### PR TITLE
Feature/forecast weather

### PR DESCRIPTION
**Pull request by rachelpr**
-Adds in forecast weather for 6 days 
-Hooks up to main search function

-Does not include isLoading state (yet)

### CODE REVIEW

**Review by MGS31 — APPROVED** [review]
From what I can see looks awesome! Good work and glad to see it allllll coming together :P

File: leafsmart-front-end/src/components/ForecastWeather.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+import "./CurrentWeather.css";
+import ForecastWeatherCard from "./ForecastWeatherCard";
+import { forecastDays } from "../helpers/selectors";
+
+const ForecastWeather = (props) => {
+  const { coordinates } = props;
+  const [forecastWeather, setForecastWeather] = useState([]);
+  const [isLoading, setIsLoading] = useState([false]);
+
+  const apiUrl = process.env.REACT_APP_WEATHERBIT_ENDPOINT_ENDPOINT;
```
Comment (b1u3too) [inline leafsmart-front-end/src/components/ForecastWeather.js:13]: Maybe a typo? ENDPOINT appears twice

File: leafsmart-front-end/src/components/ForecastWeather.js [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+          temp={forecastWeather.temp}
+          day={forecastDays()[index]}
+        />
+      );
+    });
+
+  return (
+    <main>
+      <section>
+        {isLoading ? (
```
Comment (b1u3too) [inline leafsmart-front-end/src/components/ForecastWeather.js:56]: This line's going to cause isLoading to show before a person does their search -- do we want that, or do we want to quickly fix that before merging?

**Review by b1u3too — CHANGES_REQUESTED** [review]
Looks rad and HECKIN' WELL DONE RACHEL :woman_dancing: ! Can we talk about those comments before merging?

File: leafsmart-front-end/src/components/ForecastWeather.js (hunk 1 above)
Comment (rachelpr) [reply leafsmart-front-end/src/components/ForecastWeather.js:13]: just a typo!

File: leafsmart-front-end/src/components/ForecastWeather.js (hunk 2 above)
Comment (rachelpr) [reply leafsmart-front-end/src/components/ForecastWeather.js:56]: yes let's change this!

Comment (b1u3too) [reply leafsmart-front-end/src/components/ForecastWeather.js]: Awesome, thanks!

**Review by b1u3too — APPROVED** [review]

**Pull request merged**